jenkins/system-dependencies.sh

```diff
@@ -404,7 +404,7 @@ function check_specific_xcode () {
 
 	if ! test -d "$XCODE_DEVELOPER_ROOT"; then
 		if ! test -z "$PROVISION_XCODE"; then
-			if ! test -z "$JENKINS_URL"; then
+			if ! test -z "$SYSTEM.ACCESSTOKEN"; then
```

Comment:
Weird change, what is the diff?

Reply:
This was something that Connor suggested I do. There was an error in provision dependencies and Connor said that the test was returning false because it's currently not running in a Jenkins instance and to change to check for DevOps variable